packages/web/src/pages/profile-page/MobileServerProfilePage.tsx

```diff
@@ -0,0 +1,213 @@
+import { useState } from 'react'
+
+import { getUser } from '@audius/common/src/store/cache/users/selectors'
+import { formatCount } from '@audius/common/src/utils/formatUtil'
+import BadgeArtist from '@audius/harmony/src/assets/icons/ArtistBadge.svg'
+import IconDonate from '@audius/harmony/src/assets/icons/Donate.svg'
+import IconLink from '@audius/harmony/src/assets/icons/Link.svg'
+import { Box } from '@audius/harmony/src/components/layout/Box'
+import { Flex } from '@audius/harmony/src/components/layout/Flex'
+import { Text } from '@audius/harmony/src/components/text'
+import cn from 'classnames'
+
+// import ProfilePageBadge from 'components/user-badges/ProfilePageBadge'
+import { ServerUserGeneratedText } from 'components/user-generated-text/ServerUserGeneratedText'
+import { useSelector } from 'utils/reducer'
+
+import styles from './components/mobile/ProfileHeader.module.css'
+
+const messages = {
+  tracks: 'Tracks',
+  followers: 'Followers',
+  following: 'Following',
+  playlists: 'Playlists',
+  showMore: 'Show More',
+  showLess: 'Show Less',
+  editProfile: 'Edit Profile',
+  profilePicAltText: 'User Profile Picture'
+}
+
+export type OwnProps = {
+  // userId: ID
+  userId: any
+}
+
+export const ServerProfilePage = ({ userId }: OwnProps) => {
+  const user = useSelector((state) => getUser(state, { id: userId }))
+  const [isDescriptionMinimized, setIsDescriptionMinimized] = useState(true)
```

Comment:
heads up since there is no state on server, can probably just remove logic related to this